packages/common/src/components/Footer/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled from 'styled-components';
+import MaxWidth from '../flex/MaxWidth';
+import LinkGroup from './LinkGroup';
+
+import linkGroups from './linkGroups.json';
+
+const Container = styled.div`
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+  padding-top: 5rem;
+  padding-bottom: 3rem;
+  flex-wrap: wrap;
+`;
+
+const Background = styled.div`
+  position: relative;
+  background-color: ${props => props.theme.background2.darken(0.2)};
+  padding: 1rem;
+  z-index: 5;
+`;
+
+export default () => (
```

Comment:
Make this a `FunctionComponent` and create named export instead of `default export`.

Reply:
the current `Footer.tsx` (deleted) does a `export default () => (`. If I change this to use a named export, the every component that uses this would need to be changes. That, I feel, should be out-of-scope for this PR.

Reply:
Yeah I know, but in all the #Hacktoberfest PRs we're just changing it in every component that uses the changed component so that's why I commented about named export here too.